src/index.js

```diff
@@ -1 +1,27 @@
-import './index.css';
\ No newline at end of file
+import './index.scss';
+import { displayMoviesCount } from './modules/Counters.js';
+import fetchMovies from './modules/fetchMovies.js';
+
+fetchMovies();
+
+// post reservation using fetch api
+// const postReservation = () => {
+//   fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/hmHPRyTIRRJiq4ZsoTAe/reservations', {
+//     body: JSON.stringify({
+//       item_id: '1',
+//       date_start: '2020-01-01',
+//       date_end: '2020-01-02',
+//       username: 'Test User',
+//     }),
+//     method: 'POST',
+//     mode: 'cors',
+//     headers: {
+//       'Content-Type': 'application/json',
+//     },
+//   }).then((response) => response.text())
+//     .then((data) => console.log(data));
+// };
+
+// postReservation();
```

Comment:
- [OPTIONAL] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)

Reply:
thank you very much